docs/release-notes/feature-node-selectors.rst

```diff
@@ -0,0 +1,14 @@
+:orphan:
+
+**New Features**
+
+-  Kubernetes Configuration: Allow Cluster administrators to define Determined resource pools on
+   Kubernetes using node selectors and/or affinities. Configure these settings at the default pod
+   spec level under ``task_container_defaults.cpu_pod_spec`` or
+   ``task_container_defaults.gpu_pod_spec``. This allows a single cluster to be divided into
+   multiple resource pools using node labels.
+
+-  WebUI: Allow resource pool slot counts to reflect the state of the entire cluster. Allow slot
+   counts and scheduling to respect node selectors and affinities. This impacts Determined clusters
+   deployed on Kubernetes with multiple resource pools defined in terms of node selectors and/or
+   affinities.
```

Comment:
:orphan:

**New Features**

-  Kubernetes Configuration: Allow Cluster administrators to define Determined resource pools on
   Kubernetes clusters using node selectors and/or affinities. This feature allows a single cluster to be divided into multiple resource pools using node labels. To configure resource pools, modify the default pod spec settings under ``task_container_defaults.cpu_pod_spec`` or
   ``task_container_defaults.gpu_pod_spec``. 

-  WebUI: Reflect the resource pool slot counts for the entire Kubernetes cluster. This ensures that slot counts and scheduling decisions respect the defined node selectors and affinities. This impacts Determined clusters
   deployed on Kubernetes with multiple resource pools defined using node selectors and/or affinities.